ddp-workspace/projects/toolkit/src/lib/components/app/app-redesigned-base.component.ts

```diff
@@ -39,20 +39,24 @@ export class AppRedesignedBaseComponent implements OnInit, OnDestroy {
     }
 
     private initMailingListDialogListener(): void {
-        const modalOpen = this.communicationService.openJoinDialog$.subscribe(() => {
-            this.dialog.open(JoinMailingListComponent, this.DIALOG_BASE_SETTINGS);
-        });
-        this.anchor.addNew(modalOpen);
+        if(this.communicationService) {
```

Comment:
Is this file change related to error mentioned in PR comment?

Reply:
Yes